ejcIMC.py

```diff
@@ -33,10 +51,11 @@ def whileErrorNumbers(validation):
         value = numbersValidation(heigth)
```

Comment:
I suggest change this name to heightValidation or something like that... value say nothing to reffer about that variable